### ISSUE TITLE
read / unread status

### ISSUE THREAD

**Issue by anarcat** [bookieio/Bookie#454]
so i find bookie quite interesting, but i am still hesitant between using bookie and [wallabag](http://wallabag.org/). one of the interesting features of wallabag is that it allows the user to mark posts as read or not. it also notices where the user is at when reading a specific page, which is also quite useful to read longer articles.

the first step to match that functionality would be to have a "read/unread" status per post. then the icing on top would be to track the position in the page. this would give pretty much feature parity with wallabag, i believe.

a demo of wallabag is available here: http://demo.wallabag.org/

**Comment by mitechie** [comment]
Thanks for the feedback. This is something I've long wanted to implement. With the readable parsing it makes sense to mark things as "toread" and catching up on them. I've compared the idea more to the pocket/instapaper of the world, I'll have to check out wallabag.

Towards this end, Bookie has the idea of a "command tag" that starts with a ! and the first one implemented is !toread. All this currently does is to mark a bookmark with the tag "toread". The idea is that, later on we could take these and build up email or kindle downloads of articles that get sent to users.

There's also been the idea of a different web view, especially in the responsive small phone size, that would allow swiping from article to article marked as toread. It's not something that has been started yet.

The final aspect of this, is that I've talked with the great guys doing the android app and one of their ideas was to add a "saved search" idea in the app. That would make it easy to save a search as those bookmarks tagged "toread" and then to make it easy to get the readable view on the app and mark as read by removing the tag.

So I guess the tl;dr yep, this would be great and it's something we all want and need to find the time to do.

**Comment by anarcat** [comment]
awesome.

some feedback on the "toread" bookmark idea - i've been using this in my current bookmarking software, and it's fairly limited because we need an easy way to remove that tag within the app. from a user perspective, it's not obvious this feature is accessible if there isn't a clear "button" to mark or unmark an article as read.

of course, if the !toread tag is hidden behind a nice user interface, all that doesn't really matter. :)

wallabag takes the approach of hiding those implementation details from the user and simply has a checkmark button to mark an article as read or not, and transparently remembers where you are when reading an article. for example, if you go on the demo site right now and look at [this article](http://demo.wallabag.org/index.php?view=view&id=1426), you'll see it will scroll down halfway through the page to where i stopped reading it... but i guess this is separate feature request.

**Comment by mitechie** [comment]
Yes, the idea is that commands could do things via the tags interface but get out of the way of the user. Kind of like keyboard shortcuts. For instance, one idea was to allow a tag stream of 

```
!email me@you.com github git tutorial
```

Which would apply the tags github, git, tutorial, but then send the email to the user as a side task. 

Yes though, the goal would be to provide a decent reading experience that's lacking right now.
